src/main/scala/org/broadinstitute/dsde/workbench/avram/api/AvramRoutes.scala

```diff
@@ -16,4 +25,26 @@ class AvramRoutes {
   def ping: Pong = {
     Pong()
   }
+
+  // TODO: remove this endpoint when we have more meaningful ways to test database queries
+  @ApiMethod(name = "now", httpMethod = "get", path = "now")
+  def now: Now = {
+    // Explicitly use a Future to make sure the implicit ExecutionContext is being used
```

Comment:
So `scala.concurrent.ExecutionContext.Implicits.global` works in App Engine? I was wondering if we needed to define our own single-threaded ExecutionContext.

Reply:
Also I know this is a toy API but isn't this essentially doing:
```
Await.result(Future(Await.result(database.run(...))))
```
? Not sure you need the outer `Await`/`Future`.

Reply:
My understanding of threads in app engine is that _background_ threads aren't allowed in app engine standard with automatic scaling (though _are_ allowed with basic or manual scaling, both of which do not scale to zero). Threads during requests are supported. You can use either the GAE thread API or the standard Java management API, though you can't call other GCP API calls from Java-managed threads. If we find we need to do that, that's when we'll need a custom execution context that uses the GAE thread API.

Reply:
Ah, you're right. Looks like I double wrapped it.

Reply:
Actually, explicitly referencing a `Future` here necessitates an `ExecutionContext` in a way that `Await.result()` does not. I need to understand this better to simplify it so for now I'm leaving it as-is.